docs/resources/dms_kafka_smart_connect_task.md

```diff
@@ -0,0 +1,121 @@
+  The `obs_destination_descriptor` block supports:
+
+* `access_key` - (Required, String, ForceNew) Specifies the access key of the HuaweiCloud.
+  Changing this parameter will create a new resource.
+
+* `secret_key` - (Required, String, ForceNew) Specifies the secret key of the HuaweiCloud.
+  Changing this parameter will create a new resource.
+
+* `consumer_strategy` - (Required, String, ForceNew) Specifies the consumer strategy of the smart connect task.
+  Value options:
+  + **latest**: Read the latest data.
+  + **earliest**: Read the earliest data.
+
+  Changing this parameter will create a new resource.
+
+* `deliver_time_interval` - (Required, Int, ForceNew) Specifies the deliver time interval of the smart connect task.
+  The value should be between 30 and 900. Changing this parameter will create a new resource.
+
+* `obs_bucket_name` - (Required, String, ForceNew) Specifies the obs bucket name of the smart connect task.
+  Changing this parameter will create a new resource.
+  
+* `partition_format` - (Required, String, ForceNew) Specifies the time directory format of the smart connect task.
+  Value options: **yyyy**, **yyyy/MM**, **yyyy/MM/dd**, **yyyy/MM/dd/HH**, **yyyy/MM/dd/HH/mm**.
+  Changing this parameter will create a new resource.
+
+* `obs_path` - (Optional, String, ForceNew) Specifies the obs path of the smart connect task.
+  Obs path is separated by a slash. Changing this parameter will create a new resource.
+
+* `topics` - (Optional, String, ForceNew) Specifies the topic names separated by a comma of the smart connect task.
+  Changing this parameter will create a new resource.
+
+* `topics_regex` - (Optional, String, ForceNew) Specifies the regular expression of topic name for the smart connect task.
+  Changing this parameter will create a new resource.
+
+-> **NOTE:** Exactly one of `topics`, `topics_regex` should be specified.
+
+* `destination_file_type` - (Optional, String, ForceNew) Specifies the destination file type of the smart connect task.
+  Only **TEXT** is supported. Changing this parameter will create a new resource.
+
+* `record_delimiter` - (Optional, String, ForceNew) Specifies the record delimiter of the smart connect task.
+  Value options: **,**, **;**, **|**, **\n**, **""**. Defaults to **\n**.
+  Changing this parameter will create a new resource.
+
+## Attribute Reference
+
+In addition to all arguments above, the following attributes are exported:
+
+* `id` - The resource ID.
+
+* `topics` - Indicates the topic config of the smart connect task. There are 2 kinds of values.
+  One is topic name string separated by a comma, the other is a regular expression of topic name.
+
+* `status` - Indicates the status of the smart connect task. The value can be **RUNNING**, **PAUSED**.
+
+* `created_at` - Indicates the creation time of the smart connect task.
+
+## Import
+
+The kafka smart connect task can be imported using the kafka instance `connector_id` and `id` separated by a slash, e.g.
+
```

Comment:
as some attributes can not be imported, we should add a note here.

Reply:
done